superbench/benchmarks/micro_benchmarks/computation_communication_overlap.py

```diff
@@ -114,6 +114,21 @@ def add_parser_arguments(self):
             required=False,
             help='The number of test step.',
         )
+        self._parser.add_argument(
+            '--distributed_impl',
+            type=DistributedImpl,
+            default=DistributedImpl.DDP,
```

Comment:
do we need to set ddp as default for models?
https://github.com/microsoft/superbenchmark/blob/3d72c07807e23d572c0889faa481cfe504e947fd/superbench/benchmarks/model_benchmarks/model_base.py#L111-L114

Reply:
No. Since ddp is the only impl for these microbenchmark, but for model benchmark, the impl can be None which means single GPU.